Makefile

```diff
@@ -7,7 +7,14 @@ DESTDIR ?=
 SIGN ?=
 PREFIX ?= /usr/local
 FLAKY_TESTS ?= run
+STAGINGSERVER ?= node-www
 
+OSTYPE := $(shell uname -s | tr '[A-Z]' '[a-z]')
+
+# Flags for packaging.
+BUILD_DOWNLOAD_FLAGS ?= --download=all
```

Comment:
Perhaps not relevant to 0.10/0.12/other versions but I'd like to avoid usage of `all` since it could expand in the future (where i'd prefer it being removed). Could we use `--download=icu` instead?